tests/exceptions/HO100236.test.ts

```diff
@@ -24,10 +24,10 @@ describe("HO100233", () => {
       cjsCode: "MC8080524",
       offenceCategory: "CB",
       offenceTitle: "Application to reopen case",
-      recordableOnPnc: "N",
+      recordableOnPnc: false,
       description: "blah",
       homeOfficeClassification: "123/45",
-      notifiableToHo: "Y",
+      notifiableToHo: false,
```

Comment:
Should be `true`

Reply:
Ooops, that is not looking good

Reply:
I'll have a look

Reply:
Good catch, updated, although I found it was irrelevant for the test